fixtures/README.md

```diff
@@ -0,0 +1,36 @@
+# Fixtures
```

Comment:
I like the idea of having a readme to describe the intent of the fixtures. But it might be better if the README for each fixture was located in it's own folder and this README could link out. 

I can only imagine over time this README growing in size.

What's your thoughts?

Reply:
I am probably biased, but I think a single index-like file can be scanned quicker (with `CRTL+F`) then opening each folder separately. I also feel it might be easier for new contributors to go through a single README file to check if there is already a project fitting new scenarios they would like to cover. For more complicated projects we could have brief descriptions in the main README and links to separated READMEs (as you suggested). I hope we can limit the number of sample projects, reuse them for different commands, and enhance over time. 

Agree - README size is a valid concern, one day the file might become very long and unreadable. I just thought that when it happens, we will quickly split it up and follow README per-fixture folder pattern.

I am happy to update the PR and move the descriptions to separated README files. We can always revisit it later, shouldn't take much time to move from one convention to another.

Reply:
I'm happy with that reasoning, we can address it later if it starts becoming a problem